components/localbox/timeslot.blade.php

```diff
@@ -58,21 +58,27 @@ class="dropdown-content px-4 py-3 hide"
                     value="{{ $orderTimeIsAsap ? 'asap' : 'later' }}"
                     autocomplete="off"
                 />
-                <div class="form-group">
-                    <select
-                        class="form-control"
-                        data-timepicker-control="date"
-                        data-timepicker-label="@lang('igniter.local::default.label_date')"
-                        data-timepicker-selected="{{ $orderDateTime ? $orderDateTime->format('Y-m-d') : '' }}"
-                    ></select>
-                </div>
-                <div class="form-group">
-                    <select
-                        class="form-control"
-                        data-timepicker-control="time"
-                        data-timepicker-label="@lang('igniter.local::default.label_time')"
-                        data-timepicker-selected="{{ $orderDateTime ? $orderDateTime->format('H:i') : '' }}"
-                    ></select>
+                <div class="row">
```

Comment:
Add `no-gutters` here and replace `pr-0` classes with `pr-1` and `pl-1`, that should remove the space in between to make it look like input group. Then add `btn-block` to the set time button.

Reply:
All done